Frameworks/BlockchainCertificates/Requests/CertificateValidationRequest.swift

```diff
@@ -368,6 +486,28 @@ public class CertificateValidationRequest : CommonRequest {
     }
 }
 
+/// This is a simplified adaptation of the _getDataToHash function from
+/// https://github.com/digitalbazaar/jsonld-signatures. 
+/// This function prefixes the normalized json-ld with optional headers. Blockcerts only uses the 
+/// created date header for now.
+private func _getDataToHash(input: String, date: String) -> String {
```

Comment:
this is a json-ld signature requirement. If all goes well, this can be deleted next week.